src/common/task.cc

```diff
@@ -96,6 +103,11 @@ class TaskBuilder {
       ObjectID return_id = task_compute_return_id(task_id, i);
       returns.push_back(to_flatbuf(fbb, return_id));
     }
+    /* For actor tasks, add a dummy output. */
```

Comment:
We can probably just increase `num_return_vals` by one when we construct the task in Python, right?